buyer.go

```diff
@@ -432,13 +432,22 @@ func (t *ticketPurchaser) purchase(height int64) error {
 		}
 	}
 
+	// Check for new balance credits and update ticket queue if necessary
+	if int64(balSpendable)-atomic.LoadInt64(&glBalance) > atomic.LoadInt64(&glTicketPrice) {
+		log.Tracef("Current balance %v is greater than estimated "+
+			"balance: %v", int64(balSpendable), atomic.LoadInt64(&glBalance))
```

Comment:
Thanks, using a var to read `glBalance` only once. Also, made a note to review the atomic on it and remove it if possible. This PR doesn't deal with updating `glBalance` and `glTicketPrice` but I'll make a note to check if they must be updated together.